scripts/travisci/check_imports.py

```diff
@@ -37,13 +37,7 @@ def get_import_statement(node, lines):
 
 
 # Add file path prefixes here to ignore them.
-IGNORE_PATHS = [
-    "build/",
-    "dist/",
-    "docs/",
-    "garage/contrib/",
-    "sandbox/",
-]
+IGNORE_PATHS = ["build/", "dist/", "docs/", "garage/contrib/", "tests/"]
```

Comment:
Is there a reason this file was touched? It seems unrelated to the PR

Reply:
It's because for some reason `check_imports` claims that importing from the `tests` folder is invalid, e.g. `from tests.envs.dummy import DummyDictEnv` will throw an error.

Reply:
This affects the test I wrote to test this code located at `tests/tf/spaces/test_dict_space.py`